indico/modules/events/papers/templates/management/_paper_assignment_list.html

```diff
@@ -0,0 +1,115 @@
+{% from 'message_box.html' import message_box %}
+
+{% macro render_paper_assignment_list(event, total_entries, contribs) %}
+    {% set tz = event.display_tzinfo %}
+    {% if contribs %}
+        <form method="POST">
+            <input type="hidden" name="csrf_token" value="{{ session.csrf_token }}">
+            <table class="i-table tablesorter">
+                <thead>
+                    <tr class="i-table">
+                        <th class="i-table thin-column" data-sorter="false"></th>
+                        <th class="i-table id-column">
+                            {% trans %}ID{% endtrans %}
+                        </th>
+                        <th class="i-table title-column">
+                            {% trans %}Title{% endtrans %}
+                        </th>
+                        <th class="i-table state-column">
+                            {% trans %}State{% endtrans %}
+                        </th>
+                        <th class="i-table revision-column">
+                            {% trans %}Revision{% endtrans %}
+                        </th>
+                        <th class="i-table judge-column" data-sorter="false">
+                            {% trans %}Judges{% endtrans %}
+                        </th>
+                        <th class="i-table content-reviewers-column" data-sorter="false">
+                            {% trans %}Content reviewers{% endtrans %}
+                        </th>
+                        <th class="i-table layout-reviewers-column" data-sorter="false">
+                            {% trans %}Layout reviewers{% endtrans %}
+                        </th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for contrib in contribs %}
+                        {% set last_revision_state %}
+                            {% if contrib.paper_revisions %}
+                                {{ contrib.paper_revisions.0.state.title }}
+                            {% else %}
+                                {%- trans %}Paper not yet submitted{% endtrans -%}
```

Comment:
In some places you use `-` to remove the whitespace and in others you don't. I'd suggest to stick to one of the two conventions. The best IMO is to remove the whitespace because it creates an ugly HTML structure.